wavefront-proxy.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: wavefront-proxy
+  version: 12.4
+  epoch: 0
+  description: Wavefront Proxy Project
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-11-jre
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - openjdk-11
+      - maven
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/wavefronthq/wavefront-proxy
+      tag: proxy-${{package.version}}
+      expected-commit: 7170d26b802e5f4b2201c3a38c2d11e6e3ad719a
+
+  - runs: |
+      export LANG=en_US.UTF-8
+      export JAVA_HOME=/usr/lib/jvm/openjdk
+      mvn -f proxy --batch-mode clean package -DskipTests -DskipFormatCode
+      mkdir -p ${{targets.destdir}}/opt/wavefront/wavefront-proxy
```

Comment:
Packages should not go into `/opt`.

Reply:
the [run.sh](https://github.com/wavefrontHQ/wavefront-proxy/blob/master/docker/run.sh#L62) has a reference /opt , so we would need to override/re-write the run.sh when we do an image with a new reference ?

The only other package reference I found was from conda where the package was going to /opt

Reply:
We could do it in a compat subpackage if they hard code it.

Reply:
I moved the packages to `/usr/share/java/wavefront/wavefront-proxy` add a replace in the compat subpackage